Userland/Applications/Spreadsheet/Tests/basic.js

```diff
@@ -108,4 +108,17 @@ describe("Range", () => {
         expect(R`A0:A25`.first().name).toEqual("A0");
         expect(R`A2:A25`.first().name).toEqual("A2");
     });
+
+    test("Range#at", () => {
```

Comment:
Does JS use this notation? AFAIK it's mostly Ruby?

Reply:
java uses it too, I think its just generic knowledge that static functions are Class.Fn() and instance funcs are Class#Fn()